coralproject/talk
# Template bugs

## What does this PR do?

Fixes email template errors with the partials rewrite.

## How do I test this PR?

1. Go to http://localhost:3000/admin/confirm-email
2. Notice the error about the incorrect token instead of the template error
